tests/particle_test.cc

```diff
@@ -1223,10 +1253,23 @@ TEST_CASE("Particle is checked for 2D case", "[particle][2D]") {
         std::make_shared<mpm::Particle<Dim>>(id, coords);
 
     // Check mass
-    REQUIRE(particle->mass() == Approx(0.0).epsilon(Tolerance));
-    double mass = 100.5;
+    REQUIRE(particle->mass() == Approx(0.).epsilon(Tolerance));
+    double mass = 201.0;
```

Comment:
Same as the other one, but it's really up to you, I thought there are more fundamental reasons to change